pom.xml

```diff
@@ -101,7 +101,7 @@
         <!-- [WARNING] Using platform encoding (UTF-8 actually) to copy filtered
           resources, i.e. build is platform dependent! -->
         <version.org.jboss.as.console>1.5.4.Final</version.org.jboss.as.console>
-        <version.org.jboss.as>7.2.0.Final</version.org.jboss.as>
+        <version.org.jboss.as>7.2.0.Final</version.org.jboss.as> 
         <version.redhat.eap6.bom>6.3.0.GA</version.redhat.eap6.bom>
```

Comment:
everything is compiling fine. this is used for the libraries used on the itests. On the itests we are using eap 6.4.0